src/test/java/org/ballerinalang/client/ResponseHolder.java

```diff
@@ -0,0 +1,55 @@
+package org.ballerinalang.client;
```

Comment:
Add license header, fix in all the places.

Reply:
This is fixed with #10